packages/wells/README.md

```diff
@@ -181,8 +181,8 @@ wellbores.forEach(async wellbore => {
 ```js
 const wellboreId = 4618298167286402;
 const trajectories = await client.surveys.listTrajectories(wellboreId);
-trajectories.forEach(async trajectory => {
-  const rows = await element.rows();
+for (const trajectory of trajectories) {
+  const rows = await trajectory.rows();
   console.log(rows);
 });
```

Comment:
Since we are doing async calls, maybe instead of awaiting inside the for loop, maybe it's possible to create a promise for each item and await them all at once? I think there is something called `Promise.all()` that takes a list of futures/promises and waits until the last one is done. If the `trajectory.rows()` function call is going to do any network calls, all network calls will be done sequentially now.

Or maybe it doesn't matter. Could be an idea if we want to improve performance.